src/product-card.css

```diff
@@ -0,0 +1,87 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  max-width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  align-items: center;
+  color: #060b35;
+}
+
+.product-card__image {
+  margin: 0 0 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.product-card__title {
+  margin: 0 0 4px;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+}
+
+.product-card__code {
+  margin: 0 0 16px;
+  color: #616070;
+  font-size: 10px;
+  align-self: start;
+  line-height: 14px;
+}
+
+.product-card__reviews-wrapper {
+  margin: 0 0 24px;
+  display: flex;
+  justify-content: space-between;
+  line-height: 14px;
+  width: 100%;
+}
+
+.product-card__reviews-text {
+  margin: 0;
+  font-size: 10px;
+}
+
+.product-card__price-wrapper {
+  display: flex;
+  margin: 0 0 16px;
+  font-size: 16px;
+  line-height: 18px;
+  width: 100%;
+  justify-content: space-between;
+}
+
+.product-card__price-text {
+  margin: 0;
+  font-size: 12px;
+  color: #616070;
+}
+
+.product-card__price-price {
+  margin: 0;
+  font-weight: 700;
+}
+
+.product-card__buy-button {
+  background-color: #00acdc;
+  color: #fff;
+  width: 100%;
+  text-align: center;
+  min-height: 40px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-decoration: none;
+  text-transform: uppercase;
+  font-size: 14px;
+  font-weight: 700;
+  border-radius: 5px;
+  border: solid 1px transparent;
+}
+
+.product-card__buy-button:hover {
+  background-color: #fff;
+  color: #00acdc;
+  border-color: #00acdc;
+  transition: 0.3s ease-in-out;
```

Comment:
Its not required in this task, but better use transition on class selector without ":hover" so animation can work in both ways. Other than that i cannot see any problems.